test/benchmark/shell_scripts/iGenVar_init.sh

```diff
@@ -0,0 +1,51 @@
+# -------- -------- Prepare logfile and runtime computation -------- -------- #
+mkdir logs
+echo "Logifle written to: initial_steps.log"
+exec 3>&1 1>>logs/initial_steps.log 2>&1                                # write stdout 1,2 to logfile 3 to console
+echo $(date)
+start=$(date +%s)                                                       # get starting date
+
+# -------- -------- get & build iGenVar -------- -------- #
+mkdir Repos
+cd Repos/
```

Comment:
rather do `mkdir -p Repos && cd Repos` since you do it for other stuff like this, too?